### PR TITLE
CMS-320 form filter view

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-320

Before:

screen shot 2015-08-31 at 16 55 30

After:

screen shot 2015-09-08 at 16 39 38

### CODE REVIEW

**Comment by Brankub** [comment]
:+1:

**Comment by vxsx** [comment]
is it ok that the plus is misaligned?

File: djangocms_admin_style/sass/components/_content.scss [hunk 1]
```diff
@@ -249,6 +249,26 @@ body {
...
 // filter horizontal m2m
+.sortedm2m-container {
+    width: 80% !important;
+    .selector-filter {
+        position: relative;
+        display: flex;
+        width: 100% !important;
+        input {
+            display: flex;
+            flex: 1 0 auto;
```
Comment (vxsx) [inline djangocms_admin_style/sass/components/_content.scss]: don't really need flex with calc

File: djangocms_admin_style/sass/components/_content.scss [hunk 2]
```diff
@@ -249,6 +249,26 @@ body {
 }
 
 // filter horizontal m2m
+.sortedm2m-container {
+    width: 80% !important;
+    .selector-filter {
+        position: relative;
+        display: flex;
```
Comment (vxsx) [inline djangocms_admin_style/sass/components/_content.scss]: same

**Pull request merged**